src/components/group.ts

```diff
@@ -20,26 +17,32 @@ export module Components {
     constructor(components: Component[] = []) {
       super();
       this.classed("component-group", true);
-      components.forEach((c: Component) => this.add(c));
+      components.forEach((c: Component) => this.append(c));
+    }
+
+    protected _forEach(callback: (component: Component) => any) {
```

Comment:
Note to be a big careful as I think `callback`'s purpose is different from `ComponentCallback` and it's not obvious that this is the fact.

The thing that points this out is that `forEach` callbacks usually take in an index / the whole array but that doesn't apply at all to `ComponentCallback`s